### PR TITLE
Added get commands for orgs, repos, robots, and robot tokens

### PR DESCRIPTION
**Pull request by AlainRoy**
### Description

Added the following commands:

```
up org get ...
up repo get ...
up robot get ...
up robot token get ...
```
I have:

- [X] Read and followed Upbound's [contribution process](https://git.io/fj2m9).
- [X] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR, as appropriate.

### How has this code been tested

Manual testing, similar to other commands.

### CODE REVIEW

File: cmd/up/repository/get.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	"github.com/upbound/up/internal/upbound"
+)
+
+// AfterApply sets default values in command after assignment and validation.
+func (c *getCmd) AfterApply(kongCtx *kong.Context, upCtx *upbound.Context) error {
+	kongCtx.Bind(pterm.DefaultTable.WithWriter(kongCtx.Stdout).WithSeparator("   "))
+	return nil
+}
+
+// getCmd gets a single repo
```
Comment (hasheddan) [inline cmd/up/repository/get.go:34]:
```suggestion
// getCmd gets a single repo.
```

File: cmd/up/robot/get.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		return err
+	}
+	if a.Account.Type != accounts.AccountOrganization {
+		return errors.New(errUserAccount)
+	}
+
+	// The get command accepts a name, but the get API call takes an ID
+	// Therefore we get all robots and find the one the user requested
+	// The API doesn't guarantee uniqueness, but we just print the first
+	// one we fine. If a user wants to list all of them, they can use
```
Comment (hasheddan) [inline cmd/up/robot/get.go:54]: I’m guessing this is a copy of a typo I made elsewhere 😅
```suggestion
	// one we find. If a user wants to list all of them, they can use
```

File: cmd/up/robot/token/get.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+	"github.com/upbound/up/internal/upbound"
+)
+
+// AfterApply sets default values in command after assignment and validation.
+func (c *getCmd) AfterApply(kongCtx *kong.Context, upCtx *upbound.Context) error {
+	kongCtx.Bind(pterm.DefaultTable.WithWriter(kongCtx.Stdout).WithSeparator("   "))
+	return nil
+}
+
+// deleteCmd deletes a robot token on Upbound.
```
Comment (hasheddan) [inline cmd/up/robot/token/get.go:41]:
```suggestion
// getCmd gets a robot token on Upbound.
```

File: cmd/up/organization/get.go [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	"github.com/upbound/up/internal/upbound"
+)
+
+// AfterApply sets default values in command after assignment and validation.
+func (c *getCmd) AfterApply(kongCtx *kong.Context, upCtx *upbound.Context) error {
+	kongCtx.Bind(pterm.DefaultTable.WithWriter(kongCtx.Stdout).WithSeparator("   "))
+	return nil
+}
+
+// getCmd gets a single control plane in an account on Upbound.
```
Comment (hasheddan) [inline cmd/up/organization/get.go:35]:
```suggestion
// getCmd gets a organization on Upbound.
```

**Review by hasheddan — COMMENTED** [review]
Thanks @AlainRoy ! A few nitpicks below 🙂

**Review by tnthornton — COMMENTED** [review]
Thanks @AlainRoy! I left few comments about code style preferences  (along with the reference) and a nit or two. Otherwise LGTM 👍

**Review by tnthornton — APPROVED** [review]

**Pull request merged**